standards-catalogue/Rakefile

```diff
@@ -68,9 +68,19 @@ namespace :schema do
       end
       raise unless errors.all?
     end
+
+    desc "Validate tags are schema compliant"
+    task :tags do
+      schema = schema("tag")
+      tags = YAML.load_file("data/tags.yml")
+      errors = tags.map do |_, tag|
+        is_valid_schema(schema, tag)
+      end
```

Comment:
:bell: ding ding ding! We've hit the third time we've duplicated htis code, we should look at this being a common method

Reply:
done in https://github.com/alphagov/data-standards-authority/pull/127/commits/092e3f44c856b987a821d7c39c570dfc9afcde34